src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -553,6 +553,25 @@ public BlittableJsonReaderObject ReadForMemory(Stream stream, string documentId)
             return ReadObjectInternal(builder, documentId, mode, modifier);
         }
 
+        public BlittableJsonReaderArray ReadArray(DynamicJsonArray builder, string documentId,
```

Comment:
Why do you need this? We almost never get or send arrays directly, to have better versionning support